crates/ty/docs/configuration.md

```diff
@@ -162,17 +162,17 @@ typeshed = "/path/to/custom/typeshed"
 
 #### `root`
 
-The root(s) of the project, used for finding first-party modules.
+The root of the project, used for finding first-party modules.
 
 **Default value**: `[".", "./src"]`
```

Comment:
Hmm, it's weird that we default to multiple values if it can't set it to multiple values

Reply:
Yeah, it is. Not sure how to fix.

Reply:
Maybe we could say something like

```suggestion
**If not set, two first-party roots are inferred by default: `"."` and `"./src"`**
```

?